<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zork</class>
 <widget class="QMainWindow" name="Zork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>547</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MV Boli</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Zork</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>MV Boli</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="overwriteMode">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="imageBackground">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>240</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/resources.qrc">:/new/images/images/gym-room.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="putButton">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Put</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="eastButton">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>East</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="infoButton">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Info</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="quitButton">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="southButton">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>South</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="mapButton">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Map</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="westButton">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>West</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="takeButton">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Take</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="northButton">
        <property name="font">
         <font>
          <family>MV Boli</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>North</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
